src/event/global_events.js

```diff
@@ -134,7 +127,7 @@ module.exports = {
   validateSelector: eventsCore.validateSelector,
   bindVirtualEvent: bindVirtualEvent,
   unbindVirtualEvent: unbindVirtualEvent,
-  registerEventHandler: registerEventPlugin,
+  registerEventHandler: registerEventHandler,
```

Comment:
This function had 3 internal names.  Now it has 2.  I'd like to standardize on 1... either `addEventPlugin` or `registerEventHandler`.

Any option @andrewrota @MattDeeg?